pkg/apic/apiserver/models/api/v1/resourcemeta.go

```diff
@@ -57,6 +66,31 @@ func (rm *ResourceMeta) GetMetadata() Metadata {
 	return rm.Metadata
 }
 
+// GetSelfLink gets the resource metadata selflink
+func (rm *ResourceMeta) GetSelfLink() string {
+	if rm != nil {
```

Comment:
If rm is nil it will skip trying to find the SelfLink, and then go to the next if which checks `rm.Group`, and we will hit a nil pointer. If you want to check for nil It should return early and skip the rest of the func.

Reply:
Thanks